src/objects/object.py

```diff
@@ -26,13 +26,8 @@ def __init__(self, position: Point, scene: 'Scene') -> None:
         # Log creation of the `Object`
         print(self)
 
-    def occupy_field(self, position: Point) -> bool:
```

Comment:
Please, don't remove these. Right now they're used in `Destroy` and will be used in the future. Refactor them if they do not suit your changes.

Reply:
Sure thing, forgot that one. I found that later in code, but haven't got the time to fix. The reason is I just searched for `occupy_field` and `Destroy` uses `free_field`. It should be done same way as in `Move`. Generally, 

I think that the first idea of `occupy`+`free` was wrong because it requires `Object` implementations to have too much knowledge of scene. We should watch out in the future for such mistakes.

Reply:
Restored